physionet-django/physionet/settings/production.py

```diff
@@ -1,68 +0,0 @@
-import os
```

Comment:
Overall we agree that moving to a single `settings.py` module makes sense. 

It would be helpful to keep the original staging.py and production.py modules temporarily to give us time to switch the servers to the new configuration. Is this possible?

Reply:
Yes, I'll bring them back